arangod/Aql/Functions.cpp

```diff
@@ -7523,3 +7524,85 @@ AqlValue Functions::NotImplemented(ExpressionContext* expressionContext,
   registerError(expressionContext, "UNKNOWN", TRI_ERROR_NOT_IMPLEMENTED);
   return AqlValue(AqlValueHintNull());
 }
+
+AqlValue Functions::GetValidation(ExpressionContext* expressionContext,
+                              transaction::Methods* trx,
+                              VPackFunctionParameters const& parameters) {
+
+  static char const* AFN = "GET_SCHEMA"; // GET_VALIDATON("collectionName")
+
+  if(parameters.size() != 1) {
```

Comment:
The whitespace style differs on a line-by-line basis:
```
 if(parameters.size() != 1) {
 if (collectionName.empty()){
  if(!logicalCollection) {
```
Can we please adopt the ArangoDB coding conventions, which would mean
```
 if (parameters.size() != 1) {
 if (collectionName.empty()) {
  if (!logicalCollection) {
```
Here and everywhere else.